### DISCUSSION TITLE
N64DD compatibility

### DISCUSSION THREAD

**Discussion by Blurrypinkfloydian** [mupen64plus-ae/mupen64plus-ae#1096] [Q&A]
Although it wouldn't be huge bummer to not be able to play these games i wanna know if it's possible to play N64DD and if so what's the process, i know i probably need to add the N64DD BIOS first and foremost if it's compatible, if it is compatible what folder do i add the BIOS file and what name shall i rename it to?

**Comment by Blurrypinkfloydian** [comment]
I am a android user btw

**Comment by fzurita** [comment]
Most games work but there are still issues. Compatibility is better if you use the Angrylion RDP plus video plugin (Software renderer).

You can add your 64DD IPL under settings then data.

**Reply by Blurrypinkfloydian** [reply]
In case of the N64DD IPL do i have to add a BIOS or is do i add the folder from where N64DD games will be pulled from?

**Reply by Blurrypinkfloydian** [reply]
Ok nvm i understood how to do it, it finally recognized N64DD games, they didn't have a cover image(tho TBF idk if the emulator has them for N64DD) but the main issue is that when i tried playing them all i got was a 3d mupen 64 logo

**Reply by fzurita** [reply]
You will probably need to change the emulation profile. Try using GLideN64-Very-Accurate or the Software renderer.

**Reply by Blurrypinkfloydian** [reply]
Yet again a mupen64 logo bouncing around the screen like the DVD logo is all i got

**Reply by Blurrypinkfloydian** [reply]
Is this a issue you know of or could this be a result of the ROMs I'm using?

**Reply by fzurita** [reply]
Which 64DD game are you trying to play?

**Reply by Blurrypinkfloydian** [reply]
I tried the Mario artist games and doshin the giant

**Reply by fzurita** [reply]
Are you sure you have the correct Japanese 64DD IPL selected under settings then data.

**Reply by Blurrypinkfloydian** [reply]
Hold on

**Reply by Blurrypinkfloydian** [reply]
I'll update if anything changes

**Reply by Blurrypinkfloydian** [reply]
Ok I'm pretty sure I used the correct file, i did pull it up from my SD card tho, do i need to add it to some folder?

**Reply by fzurita** [reply]
No, just select the file under settings then data.
